src/main/java/duke/model/Model.java

```diff
@@ -13,6 +15,7 @@
 import duke.model.transports.BusService;
 import duke.model.locations.BusStop;
 import duke.model.locations.Venue;
+import duke.model.transports.Route;
 
 import java.util.List;
 
```

Comment:
No idea why Github is showing me the old model here, but please check if ur side is the updated one

Reply:
Cross-checked with current build. This PR contains an updated one.